src/components/TheHeader.vue

```diff
@@ -83,6 +83,11 @@ const linkColor = computed(() => variant.value === 'light' ? 'text-surface-1' :
 const logoColor = computed(() => variant.value === 'light' ? 'brown' : 'white');
 
 const tabColor = (page) => {
+
+  if(route.name === 'SpectateList') {
+    return page === 'Home' ? 'text-newPrimary' : linkColor.value;
+  }
```

Comment:
This feels a bit awkward, though I see why you're doing it. I think we may be stuck with this if we go the separate routes approach.